### DISCUSSION TITLE
绑定数据迁移问题

### DISCUSSION THREAD

**Discussion by shanananana** [Arondight/Adachi-BOT#825] [Q&A] [answered]
服务器到期 迁移新服务器后。bot使用正常。希望把老的用户绑定数据迁移过去。scp data/db文件夹后。发现历史绑定数据在新服务器的map.json中可以查到。但是使用深渊、米游社等功能。提示未绑定米游社信息。是有其他文件遗漏了吗

**Comment by Arondight** [comment, marked as answer]
1. 停掉新 bot
2. scp 旧数据
3. 启动新 bot

**Reply by shanananana** [reply]
旧数据指哪个目录呢？data下所有吗？

**Reply by Arondight** [reply]
> 旧数据指哪个目录呢？data下所有吗？

可以
